src/tests/admin/conferenceRoom/ResourceQuantityIsDisplayedInTheResourceAssociatedColumnInRoomGrid.java

```diff
@@ -0,0 +1,81 @@
+package tests.admin.conferenceRoom;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomInfoPage;
+import framework.pages.admin.conferencerooms.RoomResourceAssociationsPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+import framework.utils.readers.JsonReader;
+
+/**
+ * TC08: Verify that resources added are listed in the top of room list pane
+ * TC27: Verify that resource quantity and icon are displayed as columns in Room grid.
+ * TC32: Verify that resources added are inserted as columns in Room grid. 
+ * @author Juan Carlos Guevara
+ */
+public class ResourceQuantityIsDisplayedInTheResourceAssociatedColumnInRoomGrid {
```

Comment:
The name of this test case is specific for TC27. I suggest to change it to ResourcesAreDisplayedAtTheTopOfRoomListPaneAndInRoomGrid